### DISCUSSION TITLE
New documentation format using Read the Docs

### DISCUSSION THREAD

**Discussion by set-soft** [INTI-CMNB/KiBot#487] [Announcements]
Hi All!

I moved the old monolithic markdown doc to a more modular [reST](https://docutils.sourceforge.io/rst.html) doc. The new docs are rendered by [Sphinx](https://www.sphinx-doc.org/) using the [Read the Docs](https://about.readthedocs.com/) theme and hosting:

https://kibot.readthedocs.io/en/latest/

I think this is easier to navigate and search.

One interesting thing about Read the Docs is that it supports versions, so we will be able to keep docs for different KiBot versions. This is something I think we can't do using GitHub pages. At least not in a simple and efficient way.

The drawback is that Read the Docs uses ads, but they aren't intrusive.

Comments are welcome.

**Comment by hpopols** [comment]
Hi @set-soft,

I spent tens of hours on the Reamed.md the last months without knowing there was a readthedocs.io page. I discovered it by chance reading one of your issue comments.

I think it would be nice to advertise it more with a link at the very beginning of the Readme.md

**Reply by set-soft** [reply]
The *Read the Docs* new documentation currently documents 1.6.4, not yet released.
When 1.6.4 is released it will be the default documentation. 
You can see how 1.6.4 will look going to the *dev* branch, [here](https://github.com/INTI-CMNB/KiBot/tree/dev)

**Reply by hpopols** [reply]
Very nice!!! 👍
